### PR TITLE
Style message display dismiss links.

### PR DESCRIPTION
**Pull request by gauthierm**
https://www.pivotaltracker.com/story/show/88714326

### CODE REVIEW

**Comment by sgarrity** [comment]
There's an at2x retina image here in the PR, but it doesn't appear to be used. Other than that, this change looks fine.

**Comment by gauthierm** [comment]
Retina styles added.

File: www/styles/admin-swat-local.css [hunk 1]
```diff
@@ -948,4 +956,10 @@ div.swat-progress-bar-empty {
 		background-size: 22px 22px;
 	}
 
+	.swat-message-display-dismiss-link,
+	.swat-message-display-dismiss-link:link,
+	.swat-message-display-dismiss-link:visited {
+		background: url(../images/admin-message-display-dismiss@2x.png);
+		background-size: 18px 18px;
```
Comment (sgarrity) [inline www/styles/admin-swat-local.css]: The background-size values have to include the full height of the entire image sprite, not just the portion that is displayed. As it stands, this squishes both normal and hover versions of the "X" image into 18px of height.

**Comment by sgarrity** [comment]
Hover not working in retina mode.

**Pull request merged**